exercises/word-count/examples/src/WordCount.purs

```diff
@@ -0,0 +1,32 @@
+module WordCount
+  ( wordCount
+  ) where
+
+import Control.Category ((>>>))
```

Comment:
Thanks Phil, amended & pushed.

As a side note, Travis builds are failing. https://travis-ci.org/exercism/xpurescript/builds/209606731. The message is:

`The job exceeded the maximum time limit for jobs, and has been terminated.`

The above build took ~50m. Chatted on gitter, I'll raise an issue here regarding to this - looks like we'll need to cache builds.